swerve100/src/main/java/frc/robot/autonomous/HolonomicDriveController2.java

```diff
@@ -31,7 +31,7 @@ public class HolonomicDriveController2 {
   private Rotation2d m_rotationError = new Rotation2d();
   private Pose2d m_poseTolerance = new Pose2d();
   private boolean m_enabled = true;
-  private SwerveDriveSubsystem m_robotDrive;
+  private AHRSClass m_gyro;
```

Comment:
you should either use a field (in which case you should pass a reference in the constructor and assign it to this field), or you should pass the gyro into each "calculate" call, but you shouldn't do both.